README.md

```diff
@@ -85,12 +97,39 @@ Retrieve properties of a certificate from certificate store and returns output i
 certificate_name = 'GlobalSign Root CA'
 Win32::Certstore.open("Root") do |store|
     store.retrieve(certificate_name)
+    store.close
 end
 ```
     or
 ```
 store = Win32::Certstore.open("Root")
 store.retrieve(certificate_name)
+store.close
+```
+
+### Perform multiple operations
```

Comment:
@piyushawasthi according to me we don't need to include this in readme. We can have this explained the examples we have given above. @btm please make me correct if I am wrong here.

Reply:
Yeah, I don't think we need to be so explicit about being able to call multiple methods against an object. I would expect that. But you probably want to show that you can pass a block as well.

In the case of a block you shouldn't need to close explicitly either, as there is no object returned to the user to continue using.

```
# when we return an object, the user needs to close when they're done, because we don't know when they're done.
store = Win32::Certstore.open("Root")
store.add(certificate_file_path)
store.list
store.close
```

```
# when we pass a block, the store should close itself
Win32::Certstore.open("Root") do |store|
  store.retrieve(certificate_name)
end
```